container.go

```diff
@@ -49,8 +54,9 @@ type ContainerRequest struct {
 	BindMounts   map[string]string
 	RegistryCred string
 	WaitingFor   wait.Strategy
-	Name         string // for specifying container name
-	Privileged   bool   // for starting privileged container
+	Name         string   // for specifying container name
+	Privileged   bool     // for starting privileged container
+	Networks     []string // for specifying network names
```

Comment:
I would suggest adding the network aliases too

Reply:
Addresses here: https://github.com/testcontainers/testcontainers-go/pull/76/commits/f85faef1f07e90b0f410423570782cb794669a55

I used a `map[string][]string`, so that the map indexes each network alongside an array for the aliases for the container in that network

Reply:
Aliases are set "per container", not "per network", there is no need to have a Map for them.

See
https://github.com/testcontainers/testcontainers-java/blob/c094a034f733215bfd88d41538089699caab1616/core/src/main/java/org/testcontainers/containers/GenericContainer.java#L596

Reply:
In this PR I'm setting the network aliases `per container`, so a container in the network A could have a few aliases, and in the network B could have other aliases.